ToGather/ToGather/Screens/FriendAddition/ViewModel/FriendAdditionViewModel.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  FriendAdditionViewModel.swift
+//  ToGather
+//
+//  Created by yudonlee on 2022/07/20.
+//
+
+import Foundation
+
+final class FriendAdditionViewModel: ObservableObject {
+    @Published private var friendUids: [String] = []
+    @Published private var friendNicknames: [String] = []
+    
+    func isFriendEmpty() -> Bool {
+        if friendUids.isEmpty {
+            return true
+        }
+        return false
+    }
+    func getFriendNicknames() -> [String]? {
+        if friendNicknames.isEmpty {
+            return nil
+        }
+        return friendNicknames
+    }
+    
+    func getFriendUids() -> [String]? {
+        if friendUids.isEmpty {
+            return nil
+        }
+        return friendUids
+    }
+    
+    private func appendList(uid: String, nickname: String) {
+        
+        if let _ = friendNicknames.firstIndex(of: nickname) {
```

Comment:
```suggestion
        if friendNicknames.firstIndex(of: nickname) != nil {
```
옵셔널 바인딩한 값을 사용하지 않을 경우 해당 표현이 더 좋다고 합니다!

Reply:
[반영 했습니다.](https://github.com/DeveloperAcademy-POSTECH/toGather/pull/117/commits/52f6ff7a7692173974ce66f36359b9246e38adec#diff-fcb9ffc23e02a606040626611681024ecd17fbdcc5d3e744b33e0decd4642cf5L36)
이 [사이트](https://stackoverflow.com/questions/29322977/whats-the-difference-between-if-nil-optional-and-if-let-optional)를 보고 동의해서 바꿨는데 어디서 참고했는지 알수있나요~

Reply:
swfitlint에서 warning이 떠서 참고했습니다!